File: src/main/java/com/itmo/microservices/demo/payment/api/model/PaymentSubmissionDto.java
package com.itmo.microservices.demo.payment.api.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.UUID;

@Builder
@NoArgsConstructor
@AllArgsConstructor
@Data
public class PaymentSubmissionDto {
    private LocalDateTime timestamp;
    private UUID transactionID;
}

File: src/main/kotlin/com/itmo/microservices/demo/payments/api/model/PaymentSubmissionDto.kt
package com.itmo.microservices.demo.payments.api.model

import com.fasterxml.jackson.annotation.JsonInclude
import java.util.*

@JsonInclude(JsonInclude.Include.NON_NULL)
data class PaymentSubmissionDto(
    val transactionId: UUID?,
    val timestamp: Long?
)
